docs/faq.md

```diff
@@ -0,0 +1,72 @@
+# Frequently Asked Questions
+
+This page describes the most frequently asked questions about the Scorecard use, checks or relevance. These answers are continually changing. If you know any other question that could be interesting to be covered here, please [contribute](../CONTRIBUTING.md)!
+
+## Installation / Usage
+  - [Can I have a preview of the score?](#can-i-have-a-preview-of-the-score)
+  - [What is the difference between Scorecard and other Code Scanning Tools?](#what-is-the-difference-between-scorecard-and-other-code-scanning-tools)
+
+## Check-Specific Questions
+  - [Binary-Artifacts: Is it possible to set up a blocklist to check Binary-Artifacts?](#binary-artifacts-is-it-possible-to-set-up-a-blocklist-to-check-binary-artifacts)
+  - [Code-Review: Can I set Code-Review check to ignore bot commits?](#code-review-can-i-set-code-review-check-to-ignore-bot-commits)
+  - [Fuzzing: Scorecard accepts custom fuzzers and libfuzzer?](#fuzzing-scorecard-accepts-custom-fuzzers-and-libfuzzer)
+  - [Pinned-Dependencies: Will the scorecard see not-pinned dependencies in tests with Dockerfiles?](#pinned-dependencies-will-the-scorecard-see-not-pinned-dependencies-in-tests-with-dockerfiles)
+  - [Pinned-Dependencies: Can I use version pinning instead of hash pinning?](#pinned-dependencies-can-i-use-version-pinning-instead-of-hash-pinning)
+  - [Signed-Releases: Why would I sign releases?](#signed-releases-why-would-i-sign-releases)
+
+________________________________________________________________________________
+________________________________________________________________________________
+
+## Installation / Usage
+
+### Can I have a preview of the score?
+
+Yes, a preview of the Scorecard scores can be seen at https://api.securityscorecards.dev/projects/github.com/<username_or_org>/<repository_name>/ for the repositories tracked by the Scorecard Project for being considered relevant in the Open Source scenario.
+
+Anyhow, you can generate the scores for any public repository through the CLI following the steps bellow:
```

Comment:
"You can also use the CLI to generate scores for any public repository by following these steps:"